src/python/pants/init/target_roots_calculator.py

```diff
@@ -127,23 +127,38 @@ def create(cls, options, session, symbol_table, build_root=None):
     # initialization paths.
     changed_options = options.for_scope('changed')
     changed_request = ChangedRequest.from_options(changed_options)
-
+    owned_files = options.for_global_scope().owner_of
```

Comment:
I'd like to see a comment here, following the pattern for spec_roots and changed_options, and adding the blank line before and after.